server/routes/search/handlers/search.ts

```diff
@@ -25,10 +25,15 @@ export default class SearchRoutes {
       res.locals.user,
       res.locals.user.roles,
     )
+
     const childOffences =
       !offence.isChild && (await this.offenceService.getOffencesByIds(offence.childOffenceIds, res.locals.user))
-    const parentOffence =
-      offence.isChild && (await this.offenceService.getOffenceById(offence.parentOffenceId, res.locals.user))
+
+    let parentOffence
```

Comment:
if possible we try to avoid using let - instead try to use const, eg you can replace the below with this
const parentOffence = offence?.parentOffenceId && offence.isChild 
  ? await this.offenceService.getOffenceById(offence.parentOffenceId, res.locals.user) 
  : undefined;